supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -280,7 +280,7 @@
 [discrete]
 [[grub]]
 ==== image:images/yes.png[yes] GRUB (noun)
-*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. Note that GRUB 2 has replaced what was formerly known as GRUB, which was version 0.9x. GRUB version 0.9x is now known as _GRUB Legacy_.
+*Description*: _GRUB_ is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. Note that GRUB 2 has replaced what was formerly known as GRUB, which was version 0.9x. GRUB version 0.9x is now known as "GRUB Legacy".
```

Comment:
"GRUB Legacy". > `GRUB Legacy`

Reply:
Not sure why GRUB Legacy should be marked up as a literal rather than in quotation marks or nothing at all. The other references to grub have no quotation marks or backticks.